cmd/minikube/cmd/mount.go

```diff
@@ -239,11 +239,14 @@ var mountCmd = &cobra.Command{
 		}()
 
 		err = cluster.Mount(co.CP.Runner, ip.String(), vmPath, cfg, pid)
+
 		if err != nil {
 			if rtErr, ok := err.(*cluster.MountError); ok && rtErr.ErrorType == cluster.MountErrorConnect {
 				exit.Error(reason.GuestMountCouldNotConnect, "mount could not connect", rtErr)
 			}
-			exit.Error(reason.GuestMount, "mount failed", err)
+			out.ErrT(style.Failure, "Failed mount: {{.error}}", out.V{"error": err})
+			exit.Message(reason.GuestMount, "mount failed, does host support filesystem type 9p?")
```

Comment:
@xcarolan I dont think this PR is actually doing any detection, or any improvement over the current situation,
if I am wrong, do you mind pasting the output Before and After this PR to prove?

there should be a mechanisim to Detect if this the host supports 9P or not, and if it doesn't it should detect it let the user know befrehand